Services/DataX.Metrics/DataX.Metrics.Ingestor/Ingestor.cs

```diff
@@ -133,8 +133,8 @@ private async Task StopListenerAsync(EventProcessorHost eventProcessorHost, Canc
         private async Task<EventProcessorHost> InitalizeEventProcessorHostAsync()
         {
             // start listening to the event hub 
-            var eventHubName = Utility.GetConfigValue("EventHubName");
-            var consumerGroup = Utility.GetConfigValue("ConsumerGroupName");
+            var eventHubName = ServiceFabricUtil.GetServiceFabricConfigSetting("EventHubName").Result?.ToString();
```

Comment:
We will abstract this later?  perhaps a todo to capture

Reply:
Kubernetes changes for Metrics will come later.